com.unity.render-pipelines.universal/Runtime/DeferredRenderer.cs

```diff
@@ -71,6 +71,8 @@ public sealed class DeferredRenderer : ScriptableRenderer
         Material m_TileDeferredMaterial;
         Material m_StencilDeferredMaterial;
 
+        public bool AccurateGbufferNormals { get { return m_DeferredLights.AccurateGbufferNormals; } }
```

Comment:
add public API docs for any new public API, or make it internal.

Reply:
@phi-lira I made the function internal.